src/components/Navbar.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { BsToggleOff, BsBell } from "react-icons/bs";
+import { MdMessage } from "react-icons/md";
+import "./Navbar.css";
+
+function Navbar() {
+  return (
+    <div className="Navbar mx-auto bg-indigo-100 p-5">
+      <nav className="flex justify-between  px-10">
+        <div className="flex flex-row">
+          <a className="pr-5" href="#">
+            Logo
+          </a>
+          <form>
+            <input
+              type="text"
+              id="new-todo-input"
+              className="input input__lg"
+              name="text"
+              autoComplete="off"
+              size="30"
+            />
+          </form>
+        </div>
+        <div className="flex flex-row">
+          <ul className="flex flex-row">
+            <li className="pr-5">
+              <a> Sadie </a>
+            </li>
+            <li className="pr-5">
+              <a> Home </a>
+            </li>
+            <li className="pr-5">
+              <a> Create </a>
+            </li>
```

Comment:
Let's remove these anchor tags (`<a></a>`) since it's not a good practice to use them in react. 
Using `<Link>` component which comes with `react-router` is the standard way of redirecting. 
Let's just leave the text for now.

Reply:
Oh I see. I'll have it fixed